tree/ntuple/v7/src/RField.cxx

```diff
@@ -716,14 +718,15 @@ void ROOT::Experimental::RField<std::string>::GenerateColumnsImpl(const RNTupleD
    GenerateColumnsImpl();
 }
 
-void ROOT::Experimental::RField<std::string>::AppendImpl(const ROOT::Experimental::Detail::RFieldValue& value)
+std::size_t ROOT::Experimental::RField<std::string>::AppendImpl(const ROOT::Experimental::Detail::RFieldValue& value)
 {
    auto typedValue = value.Get<std::string>();
    auto length = typedValue->length();
    Detail::RColumnElement<char> elemChars(const_cast<char*>(typedValue->data()));
    fColumns[1]->AppendV(elemChars, length);
    fIndex += length;
    fColumns[0]->Append(fElemIndex);
+   return length + sizeof(ClusterSize_t);
```

Comment:
Consider here and in all other occurrences:
```suggestion
   return length + sizeof(fElemIndex);
```
As you're really storing data members, being explicit about whose size you're talking about is nice. As is I need to do the mental effort of identifying `ClusterSize_t` with `fElemIndex` (at least my assumption is that they are correlated). It also helps with code evolution, if `fElemIndex` will ever become of type `ElemIndex_t` instead.

Reply:
Fixed in [d6928ff](https://github.com/root-project/root/pull/8703/commits/d6928ff131687ce73dcae3684ee5aad902fe97da)